### DISCUSSION TITLE
fheroes1

### DISCUSSION THREAD

**Discussion by racemus** [ihhub/fheroes2#9109] [General]
### Your question

Hi everyone,

I wonder if there is a chance to run Heroes of Might and Magic: A Strategic Quest on the fheroes2 engine?
Sorry if was discussed before, I haven't found

**Comment by oleg-derevenetz** [comment]
Hi @racemus

> I wonder if there is a chance to run Heroes of Might and Magic: A Strategic Quest on the fheroes2 engine?

If I'm not mistaken, the "Heroes of Might and Magic: A Strategic Quest" is the full title of HoMM I, while fheroes2 is an engine for HoMM II. These are games from the same series, but different, differing from each other in about the same way as HoMM III (Restoration of Erathia, Armageddon's Blade, etc, etc) differs from HoMM II. So no, it won't work.

**Comment by racemus** [comment]
@oleg-derevenetz, HoMM 1 & 2 are very similar, unlike when compared to HoMM 3. I'm sure they have the same wheels

**Comment by oleg-derevenetz** [comment]
> HoMM 1 & 2 are very similar, unlike when compared to HoMM 3. I'm sure they have the same wheels

They are no more similar than HoMM2 "similar" to HoMM3. HoMM1 has just 4 factions, there were no upgraded units, there was no concept of mana (each spell could only be used once), there were much fewer types of objects on the map than in HoMM2, etc, etc.

**Comment by kitovyj** [comment]
My dream is to make it possible to load/convert HoMMI maps in fheroes2. There are a couple of my favorite maps there("Two if by sea" for example). In principle, should not be impossible, most objects have direct analogs and the four factions of HoMMI correspond to a subset of HoMMII factions.

**Comment by Alucard648** [comment]
Or just make fheroes1 engine as side project.

**Comment by ihhub** [comment]
This is not related to fheroes2. Moving this to the discussion section.

**Comment by Matriks404** [comment]
Do not forget about fheroes4 and fheroes5 (maybe less so) as well!

That said, I think supporting some kind of subset of Heroes 2 with different logic for a lot of stuff would be feasible, isn't it? Or a fork/complete new project from scratch would be better?

**Comment by mspykerez** [comment]
I personally would sell a kidney for fheroes3 !
